doc/en/requirements.txt

```diff
@@ -9,3 +9,4 @@ sphinxcontrib-svg2pdfconverter
 # is the version that is assigned to the docs.
 # See https://github.com/pytest-dev/pytest/pull/10578#issuecomment-1348249045.
 packaging <22
+furo
```

Comment:
Should also remove the `pallets-sphinx-themes`?

Reply:
Definitely. Will remove this. + removing some extras based on https://github.com/pytest-dev/pytest/pull/9788